README.md

```diff
@@ -78,17 +78,19 @@ See [Best Practices](docs/best-practices.md).
 
 Recommended provisioner versions with Kubernetes versions
 
-| Provisioner version | K8s version   | Reason                    |
-| ------------------- | ------------- | ------------------------- |
-| [2.3.0][4]          | 1.12          | fs on block support       |
-| [2.2.0][3]          | 1.10          | Beta API default, block   |
-| [2.0.0][2]          | 1.8, 1.9      | Mount propagation         |
-| [1.0.1][1]          | 1.7           |                           |
+| Provisioner version | K8s version   | Reason                      |
+| ------------------- | ------------- | --------------------------- |
+| [2.3.1][5]          | 1.14          | local persistent volumes GA |
```

Comment:
There's no new features in 2.3.1 that depend on 1.14 is there?  Should we just update 2.3.0 to 2.3.1?

Reply:
fixed, you are right, it is version compatibility